Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md

```diff
@@ -0,0 +1,50 @@
+- **Projects Dashboard** - The first version of the Projects Dashboard has been released. Organize your Umbraco Cloud projects after preference and get a complete overview of product versions in all your environments.
+
+## [2FA authentication](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/)
+
+On Umbraco Cloud, you can in a few steps add 2-factor authentication for your Umbraco Cloud account using either On Umbraco Cloud, you can in a few steps add 2-factor authentication for your Umbraco Cloud account using either **email**, **phone**, or an **authenticator app** when logging in to the Umbraco Cloud Portal., phone, or an authenticator app when logging in to the Umbraco Cloud Portal.
+
+All you have to do to enable 2FA for Umbraco Cloud is to go to your Profile, click **Edit Profile** and choose the 2FA option.
+
+![2FA-Registration](images/2FA-Registration.gif)
+
+It can be sent to your email, via SMS to your phone, or you can use an authentication app such as Google Authenticator, Authy or similar.
+
+![Phone-auth](images/Phone-auth.png)
+
+As your Umbraco profile is used to log in to the backoffice for your projects, these are all now protected with 2FA.
+
+See the [2-factor authentication on Umbraco Cloud documentation](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/) for details. 
+
+## US Region Hosting
+
+Until now, Umbraco Cloud has been exclusively hosted in the West Europe region on Azure. Including **East US** as an option and being able to offer regional hosting to agencies, partners, and end clients that are located or work in the US is a big milestone and enabler for Umbraco Cloud.
+
+You can easily select which region you would like to host your Umbraco Cloud project while creating a new Cloud project.
+
+![2FA-Registration](images/2FA-Registration.gif)
+
+As excited as we are about making Umbraco Cloud available in the US, it is just as exciting to have the ability to expand to **new regions in the future**. If you have specific requests regarding the next region you want Umbraco Cloud to expand to, you’re welcome to let us know at product@umbraco.com.
+
+See the blog article [Umbraco Cloud expands with US regional hosting](https://umbraco.com/blog/umbraco-cloud-expands-with-us-regional-hosting/) for more information.
+
+## Projects Dashboard
+
+Psst…have you tried the new Projects Dashboard yet? The first official version of the dashboard has been released. Currently, you can find a link in the projects overview, but soon the Projects Dashboard will be the default landing page after logging into Umbraco Cloud Portal.
+
+By pressing the button “Edit groups” you are able to create project groups and with simple drag-drop functionality organize your cloud and Heartcore projects in project groups.
```

Comment:
As per the gif looks like sorting the projects more than creating project groups.

```suggestion
By clicking on “Edit groups”, you can sort projects into groups with the drag-drop functionality to organize your Cloud and Heartcore projects.
```